### PR TITLE
Creating new apply section

### PR DESCRIPTION
**Pull request by EllieNodder**
### Trello card

https://trello.com/c/0200YyKZ/5010-build-the-new-how-to-apply-section-for-the-navigation

### Context

We want to create a new section on the website to house our application content.

### CODE REVIEW

File: config/redirects.yml [hunk 1]
```diff
@@ -230,29 +229,28 @@ redirects:
...
-  "/get-help-and-support/premier-plus/get-extra-support-with-premier-plus": "/three-things-to-help-you-get-into-teaching"
-  "/premier-plus": "/three-things-to-help-you-get-into-teaching"
+  "/get-help-and-support/premier-plus/get-extra-support-with-premier-plus": "/help-and-support"
+  "/premier-plus": "/help-and-support"
   "/explore-my-options/secondary/teach-chemistry": "/train-to-be-a-teacher"
   "/return-to-teaching": "/returning-to-teaching"
   "/explore-my-options/training/university-led-undergraduate-training": "/train-to-be-a-teacher"
   "/funding-and-salary(*section)": "/funding-and-support"
   "/regional-make-a-difference-in-(*area)": "/"
-  "/how-to-apply(*method)": "/train-to-be-a-teacher/how-to-apply-for-teacher-training"
```
Comment (EllieNodder) [inline config/redirects.yml:240]: the redirects from the old /how-to-apply section on the legacy site seemed to be interfering with the build of the new /how-to-apply section - I've tweaked a lot of the redirects and have also removed this one - just want to check this all looks good to you @plisovin?

File: app/views/content/how-to-apply/subject-knowledge-enhancement.md [hunk 2]
```diff
@@ -20,6 +20,9 @@ youtube_video:
   ske-course-video:
     id: mWuHMtyFrFU
     title: Subject knowledge enhancement explained
+navigation: 30.25
+navigation_title: Subject knowledge enhancement
+navigation_description: Find out about subject knowledge enhancement (SKE) courses to help you brush up your knowledge on the subject you want to teach.
```
Comment (philspray) [inline app/views/content/how-to-apply/subject-knowledge-enhancement.md:25]:
```suggestion
navigation_description: Find out about subject knowledge enhancement (SKE) courses to help you brush up on the subject you want to teach.
```

File: app/views/content/how-to-apply/teacher-training-references.md [hunk 3]
```diff
@@ -1,13 +1,15 @@
...
-subcategory: Your teacher training application
 description: |-
   Find out which teacher training references you need to provide when you apply to train to be a teacher and what they'll need to include.
 related_content:
     Application tips from a teacher training provider : "/blog/application-tips-from-a-teacher-training-provider"
     Choosing the right teacher training course provider : "/blog/choosing-the-right-teacher-training-course-provider"
     7 things to include in your teacher training application : "/blog/7-things-to-include-in-your-teacher-training-application"
+navigation: 30.15
+navigation_title: Teacher training references
+navigation_description: Find out which teacher training references you need to provide and what they should include.
```
Comment (philspray) [inline app/views/content/how-to-apply/teacher-training-references.md:12]:
```suggestion
navigation_description: Find out who is best to give your teacher training references and what they should include.
```

File: app/views/content/train-to-be-a-teacher/initial-teacher-training.md [hunk 4]
```diff
@@ -8,7 +8,7 @@ related_content:
     Choosing the right teacher training course provider : "/blog/choosing-the-right-teacher-training-course-provider"
     What to expect on your teacher training : "/blog/what-to-expect-on-your-teacher-training"
     Training and support for early career teachers : "/support-for-early-career-teachers"
-    Your teacher training application : "/train-to-be-a-teacher/how-to-apply-for-teacher-training"
+    Your teacher training application : "/how-to-apply"
```
Comment (philspray) [inline app/views/content/train-to-be-a-teacher/initial-teacher-training.md:11]: Does this now need to be more specific? 
```suggestion
    How to apply for teacher training : "/how-to-apply"
```

**Review by philspray — COMMENTED** [review]
Looks great. I've left a few minor suggestions/comments. Feel free to ignore. :)

**Comment by EllieNodder** [comment]
> Do we want to remove or update the sub heading on /train-to-be-a-teacher/ too? It doesn't feel quite right being called 'Your teacher training application' now. Could change it to 'Experience school life' for the time being?

good point - forgot about that! will tweak

File: app/views/content/how-to-apply/if-your-application-is-unsuccessful.md [hunk 5]
```diff
@@ -5,9 +5,12 @@ description: |-
...
     Choosing the right teacher training course provider : "/blog/choosing-the-right-teacher-training-course-provider"
     7 things to include in your teacher training application : "/blog/7-things-to-include-in-your-teacher-training-application"
-    Tips for teacher training interview success : "/train-to-be-a-teacher/teacher-training-interview"
-    Your teacher training personal statement : "/train-to-be-a-teacher/teacher-training-personal-statement"
-    Your teacher training references : "/train-to-be-a-teacher/teacher-training-references"
+    Tips for teacher training interview success : "/how-to-apply/teacher-training-interview"
+    Your teacher training personal statement : "/how-to-apply/teacher-training-personal-statement"
+    Your teacher training references : "/how-to-apply/teacher-training-references"
+navigation: 30.30
+navigation_title: If your application is unsuccessful
```
[the remark this answers is not in the document]
Comment (EllieNodder) [reply app/views/content/how-to-apply/if-your-application-is-unsuccessful.md:12]: yeah, it's an issue we have on all our category pages - not sure if it's fixable @plisovin?

[the remark this answers is not in the document]
Comment (gemmadallmandfe) [reply app/views/content/how-to-apply/if-your-application-is-unsuccessful.md:12]: Given that this is a common issue and not to do with How to apply, I'll raise a separate stretch ticket on this in the backlog @plisovin @EllieNodder @philspray

File: app/views/content/how-to-apply.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
+---
+title: "How to apply for teacher training"
+heading: "How to apply"
```
Comment (gemmadallmandfe) [inline app/views/content/how-to-apply.md:3]: Is it worth having 'How to apply for teacher training' as the h1 as well as the title @EllieNodder, given that we know international qualified teachers are often confused about content relevant to them?

File: app/views/content/how-to-apply/teacher-training-application.md [hunk 7]
```diff
@@ -32,9 +31,9 @@ external_content:
     - content/train-to-be-a-teacher/promos/apply-for-your-course
 promo_content:
     - content/train-to-be-a-teacher/promos/adviser-promo-apply
-navigation: 20.25
-navigation_title: How to apply
-navigation_description: Discover tips on preparing your teacher training application, from writing your personal statement to choosing your references.
+navigation: 30.05
+navigation_title: Teacher training application
+navigation_description: Find out what goes in your teacher training application and when to apply.
```
Comment (gemmadallmandfe) [inline app/views/content/how-to-apply/teacher-training-application.md:36]: `goes in` is a bit woolly?

Should we be saying something more like `...what you need to include in...`

Comment (gemmadallmandfe) [inline app/views/content/how-to-apply/teacher-training-application.md:36]: I presume we'll look to split out the `when to apply` on a different card? Do we have a ticket in the backlog for this - might be useful to get in place before start of cycle (next sprint?)?

File: app/views/content/how-to-apply/teacher-training-interview.md [hunk 8]
```diff
@@ -1,12 +1,14 @@
...
-subcategory: Your teacher training application
 description: |-
   Prepare for a teacher training interview. Learn more about what the interview involves, what you might be asked, and how to get help to prepare.
 related_content:
-    Improve your subject knowledge : "/train-to-be-a-teacher/subject-knowledge-enhancement"
+    Improve your subject knowledge : "/how-to-apply/subject-knowledge-enhancement"
     What teacher training is like : "/train-to-be-a-teacher/initial-teacher-training"
     Choosing the right teacher training course provider : "/blog/choosing-the-right-teacher-training-course-provider" 
+navigation: 30.20
+navigation_title: Teacher training interview
```
Comment (gemmadallmandfe) [inline app/views/content/how-to-apply/teacher-training-interview.md:10]:
```suggestion
navigation_title: Teacher training interviews
```
Should we make this plural as people might have more than one interview ie with multiple providers?

File: app/views/content/how-to-apply/teacher-training-interview.md [hunk 9]
```diff
@@ -1,12 +1,14 @@
...
+navigation_description: Find out how to prepare for your teacher training interview.
```
Comment (gemmadallmandfe) [inline app/views/content/how-to-apply/teacher-training-interview.md:11]:
```suggestion
navigation_description: Find out how to prepare for your teacher training interviews.
```
Should we make this plural as people might have more than one interview ie with multiple providers?

File: app/views/content/how-to-apply/subject-knowledge-enhancement.md (hunk 2 above)
Comment (gemmadallmandfe) [inline app/views/content/how-to-apply/subject-knowledge-enhancement.md:25]: This feels too wordy - and repetitive re `knowledge` and `subject`!

File: app/views/content/how-to-apply/_questions.html.erb [hunk 10]
```diff
@@ -0,0 +1,29 @@
+<div class="category__questions__container">
+
+  <section class="container category__questions">
+
+    <div class="category__question">
+      <h2>Want to chat with an adviser?</h2>
+
+      <p>
+        Get free, one-to-one support with your application from an adviser with years of teaching experience.
```
Comment (gemmadallmandfe) [inline app/views/content/how-to-apply/_questions.html.erb:9]:
```suggestion
        Get free, one-to-one support with your application from an adviser with years of teaching experience.
```
Should we emphasise `before you apply`?

File: app/views/content/how-to-apply/_questions.html.erb [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+
+      <a href="/teacher-training-adviser/sign_up/identity" class="button">Get an adviser</a>
+    </div>
+
+    <div class="category__question">
+      <h2>Want to get face-to-face advice?</h2>
+
+      <p>
+        Attend a Get Into Teaching event, either online or in person, to get tips on your application.
+        Talk to training providers and meet others considering a career in teaching.
```
Comment (gemmadallmandfe) [inline app/views/content/how-to-apply/_questions.html.erb:21]: Not sure whether `meet others....` is so important here (but please disagree!). I wonder whether it is more something like 

`Meet training providers to find out about courses in your area and talk to teachers about what teaching is really like.`

(That's not very well written!)

File: app/views/content/how-to-apply/_why-teach.html.erb [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+        caption: "Why teach?",
+        caption_size: :l
+      ) %>
+
+      <p>
+        As a teacher, every lesson shapes a life. Find out more about what life in the classroom is really like.
+      </p>
+
+      <ul>
+        <li><%= link_to("Read teachers' stories", "/blog") %></li>
```
Comment (gemmadallmandfe) [inline app/views/content/how-to-apply/_why-teach.html.erb:16]: I'd suggest switching these links around given the dearth of blog posts?

**Review by gemmadallmandfe — COMMENTED** [review]
First batch of comments @EllieNodder !

(Looks great though)

File: app/views/content/what-is-qts.md [hunk 13]
```diff
@@ -8,7 +8,7 @@ backlink: "../../"
 related_content:
     What qualifications do you need to be a teacher? : "/is-teaching-right-for-me/qualifications-you-need-to-teach"
     Who do you want to teach? : "/train-to-be-a-teacher/who-do-you-want-to-teach"
-    How to apply for teacher training : "/train-to-be-a-teacher/how-to-apply-for-teacher-training"
+    How to apply for teacher training : "/how-to-apply/teacher-training-application"
```
Comment (gemmadallmandfe) [inline app/views/content/what-is-qts.md:11]: Should this go to the new category page?

File: app/views/content/train-to-be-a-teacher/initial-teacher-training.md [hunk 14]
```diff
@@ -8,7 +8,7 @@ related_content:
     Choosing the right teacher training course provider : "/blog/choosing-the-right-teacher-training-course-provider"
     What to expect on your teacher training : "/blog/what-to-expect-on-your-teacher-training"
     Training and support for early career teachers : "/support-for-early-career-teachers"
-    Your teacher training application : "/train-to-be-a-teacher/how-to-apply-for-teacher-training"
+    Your teacher training application : "/how-to-apply/teacher-training-application"
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/initial-teacher-training.md:11]: Would this be better as `How to apply` and going to the category page?

**Review by gemmadallmandfe — COMMENTED** [review]
Second batch of comments @EllieNodder !

File: config/redirects.yml [hunk 15]
```diff
@@ -211,16 +210,16 @@ redirects:
...
   "/premier-plus-adviser/all-the-help-you-need-to-make-a-successful-application": "/steps-to-become-a-teacher"
-  "/are-you-ready-to-apply": "/steps-to-become-a-teacher"
+  "/are-you-ready-to-apply": "/how-to-apply"
   "/whats-your-learning-style": "/train-to-be-a-teacher"
   "/overseas-applicants": "/non-uk-teachers/train-to-teach-in-england-as-an-international-student"
-  "/five-things-every-prospective-teacher-needs-to-know": "/three-things-to-help-you-get-into-teaching"
+  "/five-things-every-prospective-teacher-needs-to-know": "/steps-to-become-a-teacher"
   "/explore-my-options/return-to-teaching/return-to-teaching-in-england-from-overseas": "/non-uk-teachers/return-to-england-after-teaching-overseas"
-  "/submitting-your-application": "/train-to-be-a-teacher/how-to-apply-for-teacher-training"
+  "/submitting-your-application": "/how-to-apply/teacher-training-application"
```
Comment (gemmadallmandfe) [inline config/redirects.yml:220]: Would this be better going to the category page?

File: config/redirects.yml [hunk 16]
```diff
@@ -304,10 +302,16 @@ redirects:
   "/come-to-england-to-teach-if-you-are-a-teacher-from-outside-the-uk": "/non-uk-teachers/teach-in-england-if-you-trained-overseas"
   "/non-uk-teachers/teach-in-england-if-you-trained-outside-the-uk": "/non-uk-teachers/teach-in-england-if-you-trained-overseas"
   "/funding-your-training": "/funding-and-support"
-  "/tips-on-applying-for-teacher-training": "/train-to-be-a-teacher/how-to-apply-for-teacher-training"
+  "/tips-on-applying-for-teacher-training": "/how-to-apply/teacher-training-application"
```
Comment (gemmadallmandfe) [inline config/redirects.yml:305]: Would this be better going to the category page?

**Review by gemmadallmandfe — COMMENTED** [review]
Third (and final - for now!) batch of comments @EllieNodder

File: app/views/content/how-to-apply/_questions.html.erb (hunk 10 above)
Comment (EllieNodder) [reply app/views/content/how-to-apply/_questions.html.erb:9]: do we need 'before you apply' if we're saying 'with your application'?

File: app/views/content/non-uk-teachers.md [hunk 17]
```diff
@@ -8,7 +8,7 @@ title_paragraph: |-
 title_bg_color: white
 hero_bg_color: yellow
 hero_blend_content: true
-navigation: 50
+navigation: 40
```
[the remark this answers is not in the document]
Comment (EllieNodder) [reply app/views/content/non-uk-teachers.md:11]: oops - didn't actually mean to do that! must've got confused when I was tweaking the navigation order

File: app/views/content/how-to-apply/_questions.html.erb (hunk 10 above)
Comment (gemmadallmandfe) [reply app/views/content/how-to-apply/_questions.html.erb:9]: let's leave as is then!

File: app/views/content/train-to-be-a-teacher/get-school-experience.md [hunk 18]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "Get school experience"
 heading: "Get school experience"
-subcategory: Your teacher training application
+subcategory: Experience life in the classroom
```
[the remark this answers is not in the document]
Comment (EllieNodder) [reply app/views/content/train-to-be-a-teacher/get-school-experience.md:4]: I would say it's probs best to wait to make any further tweaks to this category page until we get to it as part of the nav work, seen as we'll be changing it anyway - what do you think @philspray

File: app/views/content/how-to-apply/teacher-training-application.md [hunk 19]
```diff
@@ -32,9 +31,9 @@ external_content:
     - content/train-to-be-a-teacher/promos/apply-for-your-course
 promo_content:
     - content/train-to-be-a-teacher/promos/adviser-promo-apply
-navigation: 20.25
-navigation_title: How to apply
-navigation_description: Discover tips on preparing your teacher training application, from writing your personal statement to choosing your references.
+navigation: 30.05
+navigation_title: Teacher training application
```
[the remark this answers is not in the document]
Comment (EllieNodder) [reply app/views/content/how-to-apply/teacher-training-application.md]: hmm feel like adding 'your' here is a bit unnecessary? Is it better to keep them more concise/simpler to scan for users? feel like the way it currently is fits in a bit more with our other category pages (i.e. 'tuition fees and maintenance loans', 'bursaries and scholarships') cc @philspray for thoughts

File: app/views/content/how-to-apply-for-teacher-training/teacher-training-interview.md [hunk 20]
```diff
@@ -1,12 +1,14 @@
...
 description: |-
   Prepare for a teacher training interview. Learn more about what the interview involves, what you might be asked, and how to get help to prepare.
 related_content:
-    Improve your subject knowledge : "/train-to-be-a-teacher/subject-knowledge-enhancement"
+    Improve your subject knowledge : "/how-to-apply-for-teacher-training/subject-knowledge-enhancement"
     What teacher training is like : "/train-to-be-a-teacher/initial-teacher-training"
     Choosing the right teacher training course provider : "/blog/choosing-the-right-teacher-training-course-provider" 
+navigation: 30.20
+navigation_title: Teacher training interviews
+navigation_description: Find out how to prepare for your teacher training interviews.
```
Comment (EllieNodder) [inline app/views/content/how-to-apply-for-teacher-training/teacher-training-interview.md:11]: @gemmadallmandfe @philspray I've tweaked the nav description so it's 'interview**s**' (plural) but it's not rendering in my review app - is it the same for you two?

Comment (philspray) [reply app/views/content/how-to-apply-for-teacher-training/teacher-training-interview.md:11]: Yes it's still 'interview' in review here.

Comment (gemmadallmandfe) [reply app/views/content/how-to-apply-for-teacher-training/teacher-training-interview.md:11]: Title has changed but not the description

File: app/views/content/how-to-apply/teacher-training-application.md (hunk 19 above)
[the remark this answers is not in the document]
Comment (gemmadallmandfe) [reply app/views/content/how-to-apply/teacher-training-application.md]: Let's leave as is then!

File: app/views/content/train-to-be-a-teacher/get-school-experience.md (hunk 18 above)
[the remark this answers is not in the document]
Comment (gemmadallmandfe) [reply app/views/content/train-to-be-a-teacher/get-school-experience.md:4]: OK - I think when internships open, we should consider adding a card then at least - will add to that specific ticket

File: app/views/content/how-to-apply-for-teacher-training/teacher-training-interview.md (hunk 20 above)
Comment (gemmadallmandfe) [reply app/views/content/how-to-apply-for-teacher-training/teacher-training-interview.md:11]: Fine now - I made a small edit to the maths page (taking a full-stop outside a link) and it seems to have pushed this change through as well

File: config/redirects.yml (hunk 1 above)
Comment (gemmadallmandfe) [reply config/redirects.yml:240]: @EllieNodder `/how-to-apply` gives a 404 now that you have changed the URL of the new section

I think you probably need to add this redirect back in again (pointing to the new section) but it will need tweaking

I suspect that the best you could do it:

"/how-to-apply": "/how-to-apply-for-teacher-training"

because if you add a wildcard, it will stop any other pages in that section being accessible?

**Comment by gemmadallmandfe** [comment]
@EllieNodder Just a thought - do we want to add a link to the new section on the home page directory?

image

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me - happy for this to be merged today as soon as it has had a tech review

**Review by plisovin — APPROVED** [review]
it looks ok, if there is an issue in the redirects.yml then it would fail the rspec test
